Include a section on chart colours (#2800)
alphagov/govuk-design-system: src/styles/colour/index.md

  {% endfor %}
 </tbody>
</table>

### Colour palette for charts

> When creating charts refer to the colour palettes and guidance set out in the Analysis Function [Data visualisation: colours guidance](https://analysisfunction.civilservice.gov.uk/policy-store/data-visualisation-colours-in-charts/).

Reviewer: ```suggestion
When creating charts, use the colour palettes and guidance set out in the Government Analysis Function [Data visualisation: colours guidance](https://analysisfunction.civilservice.gov.uk/policy-store/data-visualisation-colours-in-charts/).

The colour palettes recommended by the Government Analysis Function are based the colours shown on this page, but they've made some slight changes to improve colour contrast.
```

Reviewer: > 

```suggestion
When creating charts, use the colour palettes and guidance set out in the Government Analysis Function [Data visualisation: colours guidance](https://analysisfunction.civilservice.gov.uk/policy-store/data-visualisation-colours-in-charts/).

The colour palettes recommended by the Government Analysis Function are based on the colours shown on this page. They've made some slight changes to improve colour contrast. When all the guidance is followed, the colours mentioned in the Analysis Function colour palette will create charts which meet the Web Content Accessibility Guidelines (WCAG).
```

  {% endfor %}
 </tbody>
</table>

### Colour palette for charts
When creating charts, use the colour palettes and guidance set out in the Government Analysis Function [Data visualisation: colours guidance](https://analysisfunction.civilservice.gov.uk/policy-store/data-visualisation-colours-in-charts/).

> The colour palettes recommended by the Government Analysis Function are based on the colours shown on this page. They've made some slight changes to improve colour contrast, in line with the requirements of the Web Content Accessibility Guidelines (WCAG).

Reviewer: ```suggestion
The colour palettes recommended by the Government Analysis Function are based on the colours shown on this page. They've made some slight changes to improve colour contrast, in line with the Web Content Accessibility Guidelines (WCAG).
```

> The colour palettes recommended by the Government Analysis Function are based on the colours shown on this page. They've made some slight changes to improve colour contrast. When all the guidance is followed, the colours mentioned in the Analysis Function colour palette will create charts which meet the Web Content Accessibility Guidelines (WCAG).

Reviewer: ```suggestion
The colour palettes recommended by the Government Analysis Function are based on the colours shown on this page. They've made some slight changes to improve colour contrast, in line with the requirements of the Web Content Accessibility Guidelines (WCAG).
```
